stdlib/universe/schema_functions_test.go

```diff
@@ -876,6 +876,60 @@ func TestDropRenameKeep_Process(t *testing.T) {
 				),
 			}},
 		},
+		{
+			name: "duplicate group key",
+			spec: &universe.SchemaMutationProcedureSpec{
+				Mutations: []universe.SchemaMutation{
+					&universe.DuplicateOpSpec{
+						Column: "1a",
+						As:     "1a_1",
+					},
+				},
+			},
+			data: []flux.Table{&executetest.Table{
+				ColMeta: []flux.ColMeta{
+					{Label: "1a", Type: flux.TFloat},
+					{Label: "2a", Type: flux.TFloat},
+					{Label: "3a", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{1.0, 2.0, 3.0},
+					{1.0, 12.0, 3.0},
+					{1.0, 22.0, 3.0},
+				},
+				KeyCols:   []string{"1a", "3a"},
+				KeyValues: []interface{}{1.0, 3.0},
```

Comment:
(nit) the `keyValues` are unnecessary if you have data in the table, so omitting this line would make the test a tiny bit more concise.